Tests/Unit/TypoScript/ConvertEmailLinksImplementationTest.php

```diff
@@ -37,6 +39,12 @@ class ConvertEmailLinksImplementationTest extends \TYPO3\Flow\Tests\UnitTestCase
 	 */
 	protected $mockNode;
 
+    /**
+     * @Flow\Inject(setting="atCharReplacementString", package="Networkteam.Neos.MailObfuscator")
+     * @var array
+     */
+    protected $replacementString;
```

Comment:
For a proper test I believe it would be better to inject the value into RewriteAtCharConverter to really be able to test with different `atCharReplacementString` values.